index.html

```diff
@@ -0,0 +1,1591 @@
+                      here's how
+                    </a>
+                  </h4>
+                  <ul class="latest__list">
+                    <li class="latest__item"><a href="#">Ivan</a></li>
+                    <li class="latest__item">
+                      <!-- !! -->
+                      <time
+                        datetime="2020-07-08T17:29:19+00:00"
+                        data-full-date="Julc8, 2020 - 19:29"
+                      >
+                        1 hour ago
+                      </time>
+                    </li>
+                  </ul>
+                </div>
+              </li>
+              <!-- End Latest News Artile Block -->
+
+              <!-- Start Latest News Artile Block -->
+              <li class="col-m latest__li">
+                <a href="#" class="latest__img"></a>
+                <div class="latest__box">
+                  <h4 class="latest__heading--second forth-heading">
+                    <a href="#">
+                      Instagram now lets you pin 3 comments on your posts -
+                      here's how
+                    </a>
+                  </h4>
+                  <ul class="latest__list">
+                    <li class="latest__item"><a href="#">Ivan</a></li>
+                    <li class="latest__item">
+                      <!-- !! -->
+                      <time
+                        datetime="2020-07-08T17:29:19+00:00"
+                        data-full-date="Julc8, 2020 - 19:29"
+                      >
+                        1 hour ago
+                      </time>
+                    </li>
+                  </ul>
+                </div>
+              </li>
+              <!-- End Latest News Artile Block -->
+
+              <!-- Start Latest News Artile Block -->
+              <li class="col-m latest__li">
+                <a href="#" class="latest__img"></a>
+                <div class="latest__box">
+                  <h4 class="latest__heading--second forth-heading">
+                    <a href="#">
+                      Instagram now lets you pin 3 comments on your posts -
+                      here's how
+                    </a>
+                  </h4>
+                  <ul class="latest__list">
+                    <li class="latest__item"><a href="#">Ivan</a></li>
+                    <li class="latest__item">
+                      <!-- !! -->
+                      <time
+                        datetime="2020-07-08T17:29:19+00:00"
+                        data-full-date="Julc8, 2020 - 19:29"
+                      >
+                        1 hour ago
+                      </time>
+                    </li>
+                  </ul>
+                </div>
+              </li>
+              <!-- End Latest News Artile Block -->
+            </ul>
+          </section>
+          <!-- End The Latest News Section -->
+
+          <!-- Start The Funding Section -->
+          <section class="funding s-m-b">
```

Comment:
The latest funding section should contain 4 articles in desktop mode. Please add one more.

| Source Page	| Clone Page |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/57726348/87092216-acc93700-c258-11ea-8bb2-f802d64bf5ca.png)|![image](https://user-images.githubusercontent.com/57726348/87092150-9327ef80-c258-11ea-8581-f057958ac54d.png)|